src/content/extensions/multipleroutes/src/options.js

```diff
@@ -1,97 +1,114 @@
 'use strict';
 
+var pn = chrome.privacy.network;
+var pi = chrome.privacy.IPHandlingPolicy;
+
+var mapPolicyToRadioId = {};
+mapPolicyToRadioId[pi.DEFAULT] = 0;
+mapPolicyToRadioId[pi.DEFAULT_PUBLIC_AND_PRIVATE_INTERFACES] = 1;
+mapPolicyToRadioId[pi.DEFAULT_PUBLIC_INTERFACE_ONLY] = 2;
+mapPolicyToRadioId[pi.DISABLE_NON_PROXIED_UDP] = 3;
+
+var mapRadioIdToPolicy = {};
+if (pn.webRTCIPHandlingPolicy === undefined) {
```

Comment:
Can you make a helper method called something like browserSupportsPolicy() just to be a little more readable and use that instead of "pn.webRTCIPHandlingPolicy === undefined"?